test/e2e/infra/vagrant/push_antrea.sh

```diff
@@ -1,30 +1,44 @@
 #!/usr/bin/env bash
 
 function usage() {
-    echo "Usage: push_antrea.sh [--prometheus] [-h|--help]"
+    echo "Usage: push_antrea.sh [--prometheus] [-fc|--flow-collector] [-h|--help]
+    Push the latest Antrea image to all vagrant nodes and restart the Antrea daemons
+          --prometheus           Deploy Prometheus service to scrape metrics from Antrea Agents and Controllers
+          --flow-collector       Provide the IPFIX flow collector address to collect the flows from the Flow Aggregator service
+                                 It should be given in the format IP:port:proto. Example: 192.168.1.100:4739:udp
+                                 Please note that with this option we deploy the Flow Aggregator service along with the Antrea daemonset."
```

Comment:
maybe just "along with Antrea" - the daemonset is only a small part of all the K8s resources

Reply:
Sure.. there is a doc PR. Will address it there.